src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/configs/decider/jvm/LevelTwoActionBuilderConfig.java

```diff
@@ -0,0 +1,48 @@
+package com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.configs.decider.jvm;
+
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.configs.ConfigUtils;
+import java.util.Map;
+
+public class LevelTwoActionBuilderConfig {
+  public static final double DEFAULT_FIELD_DATA_CACHE_LOWER_BOUND = 0.02;
+  public static final double DEFAULT_SHARD_REQUEST_CACHE_LOWER_BOUND = 0.01;
+  public static final int DEFAULT_CACHE_ACTION_STEP_COUNT = 2;
+  public static final int DEFAULT_QUEUE_ACTION_STEP_COUNT = 1;
+  public static final int DEFAULT_QUEUE_BUCKET_SIZE = 10;
+  private static String FIELD_DATA_CACHE_LOWER_BOUND_CONFIG_NAME = "fielddata-cache-lower-bound";
+  private static String SHARD_REQUEST_CACHE_LOWER_BOUND_CONFIG_NAME = "shard-request-cache-lower-bound";
```

Comment:
Pls see if you really need a **lower-bound, level-1-bound, level-2-bound, upper-bound**? Can we do away with level-2 bound?